server/testutils/factories/person.ts

```diff
@@ -27,3 +28,14 @@ export const personSummaryFactory = Factory.define<PersonSummary>(() => ({
   crn: `C${faker.number.int({ min: 100000, max: 999999 })}`,
   personType: faker.helpers.arrayElement(['FullPersonSummary', 'RestrictedPersonSummary', 'UnknownPersonSummary']),
 }))
+
+export const restrictedPersonSummaryFactory = Factory.define<PersonSummary>(() => ({
+  crn: `C${faker.number.int({ min: 100000, max: 999999 })}`,
```

Comment:
May be nice to pull this out as createCRNNumber for it to be reused across factories